ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -112,12 +112,21 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) =>
         content={<div><b>Warning:</b> {warning}</div>}
       />
     }
-    {genotype.numAlt >= 0 ?
+    {variant.svType ?
+      <Header.Content>
+        {Number.isInteger(genotype.cn) &&
+        <span>CN:
+          {genotype.cn === (isHemiX ? 1 : 2) ? genotype.cn : <b><i>{genotype.cn}</i></b>}&nbsp;
+        </span>}
+        {Number.isInteger(genotype.numAlt) &&
+        <span>
+          {genotype.numAlt > 0 ? <b><i>X</i></b> : '-'}/{isHemiX || genotype.numAlt < 2 ? '-' : <b><i>X</i></b>}
+        </span>}
```

Comment:
I think it will actually look better with the X/- genotype before the CN, and with them on separate lines. Additionally, if we don't have genotype we should always show CN, but if we do have genotype then we should only show CN if its not 2 (i.e. if its an alt). After thats done, I think we need to look at a few screenshots for exome SVs that are CN only (no genotype) as well as both cases for genome SVs (ie with and without CN) to see how all the options look